apidoc/Titanium/UI/Switch.yml

```diff
@@ -118,17 +118,43 @@ properties:
 
   - name: tintColor
     summary: The color used to tint the outline of the switch when it is turned off.
+    description: |
+        The color used to tint the outline of the switch when it is turned off.
+
+        Android: Track color of the Material Switch.
     type: [String, Titanium.UI.Color]
     default: undefined
-    platforms: [iphone, ipad, macos]
-    since: "3.3.0"
+    platforms: [android, iphone, ipad]
+    since: {android: 12.4.0, iphone: 3.3.0, ipad: 3.3.0}
+
+  - name: onThumbColor
+    summary: The color used to tint the thumb icon of the switch when it is turned on.
+    description: |
+        The color used to tint the thumb icon of the switch when it is turned on.
+
+        Android: Active thumb color of the Material Switch.
+    type: [String, Titanium.UI.Color]
+    default: undefined
+    platforms: [android]
+    since: {android: 12.4.0}
+
+  - name: thumbColor
+    summary: The color used to tint the thumb icon of the switch when it is turned off.
+    description: |
```

Comment:
If no specific description is provided, you can rather onit the field to prevent duplicate content

Reply:
I removed the summery so it will just use the description as that had more info